### DISCUSSION TITLE
Multiplayer timeout/disconnection

### DISCUSSION THREAD

**Discussion by floatyy101** [ppy/osu#17697] [Q&A] [answered]
I'm usually playing lots of multiplayer in osu!stable, i notice that sometimes i have a little  lost connection icon in the lower left part of my screen, usually i'm just losing connection and reconnecting after some time without much trouble, but with lazer this seems to be a lot different, most of the times, playing multiplayer, i will get disconnected from the lobby which is quite a nuisance tbh. 
  A friend of mine tried helping me with some stuff, we went through the network logs and found a couple of errors, tried to solve by flushing dns/messing with firewall settings but it's still the same. A worth thing to notice not connected directly with lazer, but with bancho itself, would be REALLY slow download speeds in the website and using osu!direct, actually it's more like slow speed for everything i try download in my browser, not noticeable in any way in other platforms/applications
[network.log](https://github.com/ppy/osu/files/8439200/network.log)

**Comment by peppy** [comment, marked as answer]
May be the same issue being tracked at https://github.com/ppy/osu/issues/14365.
